frontend/src/components/Modal/RiderModal.tsx

```diff
@@ -55,37 +50,30 @@ const RiderModal = () => {
   useEffect(() => {
     if (isSubmitted) {
       const newRider = {
-        id: formData.id,
-        firstName: formData.firstName,
-        lastName: formData.lastName,
-        phoneNumber: formData.phoneNumber,
-        email: formData.email,
-        accessibility: formData.accessibility,
-        description: '',
-        joinDate: '',
-        pronouns: '',
-        address: formData.address,
-        favoriteLocations: [],
-        organization: '',
+        ...formData,
       };
-      fetch('/api/riders', withDefaults({
-        method: 'POST',
-        body: JSON.stringify(newRider),
-      })).then(() => {
+      fetch(
+        "/api/riders",
+        withDefaults({
+          method: "POST",
+          body: JSON.stringify(newRider),
+        })
+      ).then(() => {
         refreshRiders();
         setToast(true);
       });
       setIsSubmitted(false);
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [formData, isSubmitted, refreshRiders, withDefaults]);
 
   return (
     <>
-      {showingToast ? <Toast message='The student has been added.' /> : null}
-      <Button className={styles.addRiderButton} onClick={openModal}>+ Add Student</Button>
+      {showingToast ? <Toast message="The student has been added." /> : null}
```

Comment:
Any reason to fix double vs single quotes? I don't think you need to fix it either way, I'm more just curious.

Reply:
i think there are some weird settings in my prettier. Just turned it off.